### DISCUSSION TITLE
Welcome! How to use the new PubPub Forum.

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#1210] [General]
## 👋 Hello, PubPub Community!
  We’re using Discussions as a place to get help and connect with other members of the community. Over the next few months, we'll be replacing our Discourse forum with this one on GitHub, to give the PubPub community one place to share ideas, ask questions, talk about best practices, and get help.

You should, of course, still submit GitHub issues if you see a bug or have a specific feature request in mind. But we hope that you use this forum to:

  * Ask for support or help.
  * Share tips, tricks, and best practices.
  * Share ideas and feedback with the PubPub team and the broader community.
  * Ask any questions you’re wondering about.
  * Engage with other community members.
  * Tell the community about your Community's new content or launches.
  * Welcome others and treat everyone with respect. By participating in discussions, you agree to abide by Knowledge Futures' [Code of Conduct](https://github.com/knowledgefutures/general/blob/master/CODE_OF_CONDUCT.md).

 To get started, comment below with an introduction of yourself and tell us about how you use PubPub.

**Comment by zalpany2000** [comment]
Hi Gabriel, 

I wonder if the topic of Localization has been touched on before or not, namely Arabic content support. That is mainly the translation of the UI labels and supporting RTL. 

I searched GitHub but found no former discussions in that direction.

**Reply by gabestein** [reply]
Hello — there are a few comments on it in the forums, see e.g.: https://github.com/pubpub/pubpub/discussions/1800
https://github.com/pubpub/pubpub/discussions/1698

Unfortunately, things have not changed really since those conversations. We support Arabic content and RTL! We also try to minimize English labels in the editing and reading UI, at least. Unfortunately we don't currently have the capacity for full localization, as much as we'd like to.

**Reply by shoaib4199713-source** [reply]
"I finally tried [Bojangles' special](https://bojanglesmenus.com/bojangles-specials/) and it did not disappoint. The chicken was perfectly seasoned, and the portion size was great for the price. If you haven’t checked it out yet, you’re definitely missing out."

**Comment by hanny876** [comment]
Hi everyone! I'm currently building a resource site for Tim Hortons fans in Canada. If you're interested in accurate and up-to-date menu prices and calorie info, I’ve put together a dedicated page that might be useful: [Tim Hortons Menu Canada](https://timshortonmenu.com).
I look forward to engaging with more creators and learning from this amazing community!

**Comment by smmpanel01a** [comment]
Thanks for keeping this post honest and down-to-earth. It felt like a thoughtful moment captured in writing. We need more of that on the internet. [smm cheap](https://smmpanel.com/)

**Comment by DeborahRoberts** [comment]
I use PubPub to share and organize content for my projects and collaborate with others. Excited to connect, learn, and contribute to the community!

**Comment by donaldmatheson** [comment]
Can I ask for some advice from those who use PubPub for academic journals? I am about to submit [my journal](ethicalspace.pubpub.org) to the Directory of Open Access Journals. They've knocked us back a few times already so I'm wanting to make sure I anticipate any issues. E.g. I'm conscious we don't have a pop-up about accepting cookies, in accordance with GDPR requirements. Not sure I can do anything about that one. Anything else that's caught others out? Thanks. When/if I'm through the process, I'd be very happy to help others.

**Reply by tefkah** [reply]
hmm, strange. I also submitted journal.trialanderror.org to DOAJ, they did not mention that.

You could let them know that PubPub does not use any privacy infringing tracking, and we don't set any cookies for that purpose! So we don't need a cookie popup! Only select communities which opt into using Google Analytics have one.

**Reply by tefkah** [reply]
We only set cookies for the purpose of your logged in state, which does not require a cookie popup

**Reply by donaldmatheson** [reply]
Thanks @tefkah, that's super-useful to know. Just to clarify, the DOAJ reviewer checking our application didn't ask about a cookies popup - I'm just trying to anticipate further concerns they might have. I clearly should just hit the submit button and stop stressing :)

**Reply by tefkah** [reply]
Just go for it! And good luck!!
